app/src/main/java/org/wikipedia/history/HistoryEntry.kt

```diff
@@ -18,7 +19,9 @@ import java.util.Date
 @Serializable
 @Parcelize
 @TypeParceler<Date, DateParceler>()
-@Entity
+@Entity(
+    indices = [Index(value = ["lang", "namespace", "apiTitle"])]
```

Comment:
Not really related to this PR, but would it be possible to also add:
```
Index(value = ["authority", "lang", "apiTitle", "timestamp"])
```

for the `findEntryBy` method since this is the most used SQL query for every page when doing `createPageModel`?
```
suspend fun findEntryBy(authority: String, lang: String, apiTitle: String, timestamp: Long): HistoryEntry?
```

Reply:
That makes sense to me! Thanks.